tests/testdata/msticpyconfig.yaml

```diff
@@ -0,0 +1,39 @@
+QueryDefinitions:
+  Custom:
+    - "tests/testdata"
+    - "tests"
+TIProviders:
+  OTX:
+    Args:
+      ApiID: "123456789"
+      AuthKey: "987654321"
+    Primary: True
+    Provider: "OTX" # Explicitly name provider to override
+    LookupCategories:
+      - IPv4
+      - FileHash
+      - Public
+  VirusTotal:
+    Args:
+      ApiID:
+        KeyVaultURI: "https://scubadevsecrets.vault.azure.net/secrets/kvname/kvsecretId"
```

Comment:
might want to replace with "https://ContosoKeyVault.vault.azure.net/secrets/kvname/kvsecretId" to match other docs

Reply:
Yes. Good catch.